whatlies/embeddingset.py

```diff
@@ -446,6 +446,38 @@ def merge(self, other):
         """
         return EmbeddingSet({**self.embeddings, **other.embeddings})
 
+    def assign(self, **kwargs):
+        """
+        Adds properties to every embedding in the set based on the keyword arguments.
+
+        This is very useful for plotting because a property can be used to assign colors. This method is very
+        similar to `.add_property` but it might be more convenient when you want to assign multiple properties
+        in one single statement.
+
+        Arguments:
+            kwargs: (name, func)-pairs that describe the name of the property as well as a function on how to calculate it
+
+        Usage:
+
+        ```python
+        from whatlies.embeddingset import EmbeddingSet
+
+        foo = Embedding("foo", [0.1, 0.3, 0.10])
+        bar = Embedding("bar", [0.7, 0.2, 0.11])
+        emb = EmbeddingSet(foo, bar)
+        emb_with_property = emb.assign(dim0=lambda d: d.vector[0],
+                                       dim1=lambda d: d.vector[1],
+                                       dim2=lambda d: d.vector[2])
+        ```
+        """
+        new_set = {}
+        for k, e in self.embeddings.items():
+            new_emb = e.copy()
```

Comment:
I think this would be redundant since `add_property` returns a copy of the embedding per your recent change.